ClassLibrary/ClassLibrary/Library.cs

```diff
@@ -60,6 +54,27 @@ public Author AddAuthor(String nameAuthor,String lastNameAuthor)
             return returnAuthor;
         }
 
+        public List<CardReader> CardReaders
+        {
+            get;
+            private set;
+        }
+            = new List<CardReader>();
+
+        //I'm breaking the rule AV1115, but "AddAuthor" breaking the rule too
+        public CardReader GetCardForReader(String NameReader,String LastNameReader)
```

Comment:
there are two very different cases - Getting a reader and Searching for a reader.
you can search by name and last name, but to Get one reader you should have reader's ID.

Usually system user Searches for entity by non-unique parameters like name and then in search results finds one - and then ID of that entity is user in next steps.

bottom line: in current situation, when we don't have users, we don't need Search, only Get - meaning you should use only ID as a parameter.